projects/core/styles/components/link.less

```diff
@@ -22,8 +22,8 @@
     border: none;
     cursor: pointer;
     font: inherit;
-    color: var(--tui-text-primary);
-    text-decoration: underline solid currentColor;
+    color: inherit;
```

Comment:
I think this one we should keep, let's make it `var(--tui-text-primary);`

Reply:
fixed